parser.temper

```diff
@@ -52,19 +54,19 @@ class Parser {
   }
 
   matchRepeat(): Boolean {
-    matchChar("+") || matchChar("*") || matchChar("?")
+    matchChar(char"+") || matchChar(char"*") || matchChar(char"?")
```

Comment:
Maybe should say `char'+'` if we like that style better, but I didn't do that here.